Makefile

```diff
@@ -40,7 +40,7 @@ endif
 all: manager
 
 # Run tests
-unittest: generate fmt vet manifests
+unittest: generate fmt vet manifests validate-no-offensive-lang
```

Comment:
Should we really still keep it in the unittests then?

Reply:
Ah, nevermind I thought you'd take the GitHub Action here as well (I'd prefer it).